k9mail/src/main/java/com/fsck/k9/service/NotificationActionService.java

```diff
@@ -34,37 +35,83 @@
     private final static String EXTRA_ACCOUNT = "account";
     private final static String EXTRA_MESSAGE = "message";
     private final static String EXTRA_MESSAGE_LIST = "messages";
+    private final static String EXTRA_DONTCANCEL = "dontcancel";
```

Comment:
@MarcusWolschon: Did you introduce this to avoid canceling all notifications for an account when the action of a single notification was performed? It looks like `EXTRA_NOTIFICATION_ID` is now used for that and `EXTRA_DONTCANCEL` can be removed. Can you confirm?